contracts/token/ERC1155/ERC1155.sol

```diff
@@ -31,6 +31,9 @@ contract ERC1155 is Context, ERC165, IERC1155, IERC1155MetadataURI {
     // Used as the URI for all token types by relying on ID substition, e.g. https://token-cdn-domain/{id}.json
     string private _uri;
 
+    // Mapping token ID to that token being registered as existing (1 for existing, 0 for not existing)
+    mapping (uint256 => uint256) private _tokenExists;
```

Comment:
Why is this `uint256` instead of `bool`?

Reply:
That's because of https://github.com/OpenZeppelin/openzeppelin-contracts/pull/2185#discussion_r422364288 - do you think that's a bad idea?